### ISSUE TITLE
Please build arm64 images for all dotnet-isolated images

### ISSUE THREAD

**Issue by justinmchase** [Azure/azure-functions-docker#740]
https://mcr.microsoft.com/en-us/product/azure-functions/dotnet-isolated

**Comment by mohsinonxrm** [comment]
@justinmchase , can you give this a try:
from docker:
mohsinonxrm/azure-functions-dotnet:4-isolated6.0-appservice-arm64v8

let me know how it goes.

*Not to be used in any official or supported capacity* certainly not in production

**Comment by justinmchase** [comment]
I don't think we should be sharing links to unofficial images like this here, that image is not safe and people may be tempted to think its a work around.

**Comment by mohsinonxrm** [comment]
@justinmchase , I've updated my comment above.

**Comment by odidev** [comment]
@ [mohsinonxrm,](https://github.com/mohsinonxrm) In the Microsoft registry, I found the Azure Functions Dotnet (Isolated) image only for [amd64](https://mcr.microsoft.com/en-us/product/azure-functions/dotnet-isolated/about), so if you could release official image for arm64 it would help me to move forward in my project.  

Thank you.

**Comment by mohsinonxrm** [comment]
@odidev , I can't. This was just me trying to help the community/others like me trying to explore/experiment with ARM64 k3s cluster and trying to run Azure Functions.

Will have to wait for official Microsoft support on this. TBH, I think this should be coming by now because its been a few months now that Microsoft supports ARM64 based AKS, so I imagine MS must be on it.

**Comment by justinmchase** [comment]
Try adding a thumbs up to the original post, it will help surface this as a priority issue.

**Comment by mohsinonxrm** [comment]
I've just updated the multi-architecture images with the latest release, host and added dotnet 7 and 8-preview.

https://hub.docker.com/r/mohsinonxrm/azure-functions-dotnet/tags

mohsinonxrm/azure-functions-dotnet:4-isolated8.0-appservice-preview
mohsinonxrm/azure-functions-dotnet:4-isolated8.0-preview
mohsinonxrm/azure-functions-dotnet:4-isolated7.0-appservice
mohsinonxrm/azure-functions-dotnet:4-isolated7.0
mohsinonxrm/azure-functions-dotnet:4-isolated6.0-appservice
mohsinonxrm/azure-functions-dotnet:4-isolated6.0
mohsinonxrm/azure-functions-dotnet:4-appservice
mohsinonxrm/azure-functions-dotnet:4

_Not to be used in any official or supported capacity, certainly not in production_

**Comment by kurt-mueller-osumc** [comment]
@mohsinonxrm Is there a Dockerfile anywhere that I could look at? Perhaps in a github repo :)?

**Comment by kurt-mueller-osumc** [comment]
Now that Microsoft's released Surface Laptop, which is Windows on ARM, what's the over/under of how many months it'll be until the Azure Functions docker image for aarch64 is released?

**Comment by justinmchase** [comment]
That may do it, give it 6mo

**Comment by pinkfloydx33** [comment]
@mohsinonxrm any ETA on net9 builds?

**Comment by sipkode** [comment]
I ran into this today when I tried working with an Apple-silicon Mac user.  This is badly needed in order to make dev containers usable!

**Comment by mohsinonxrm** [comment]
> [@mohsinonxrm](https://github.com/mohsinonxrm) any ETA on net9 builds?

hmm.. yeah... let me do something about that.

**Comment by vivekjilla** [comment]
This overlaps with the arm64 request in #695; we're consolidating the arm64 asks to avoid duplicate threads. Closing as a duplicate
